packages/vue/src/vue-component-lib/overlays.ts

```diff
@@ -1,20 +1,37 @@
-import { defineComponent, h, ref, VNode } from 'vue';
+import { defineComponent, h, ref, VNode, getCurrentInstance } from 'vue';
+import { needsKebabCase } from '../utils';
 
 export interface OverlayProps {
   isOpen?: boolean;
 }
 
 export const defineOverlayContainer = <Props extends object>(name: string, componentProps: string[] = [], controller: any) => {
-  // TODO
+  /**
+   * Vue 3.0.6 fixed a bug where events on custom elements
+   * were always converted to lower case, so "ionRefresh"
+   * became "ionRefresh". We need to account for the old
```

Comment:
@liamdebeasi Shouldn't this be `became "ionrefresh"` (all lowercase) ?

Reply:
Fixed, thanks! (https://github.com/ionic-team/ionic-framework/pull/23192)